dindex_store/dindex-store-kuzu.py

```diff
@@ -0,0 +1,86 @@
+from typing import Dict
+
+import os
+import json
+import kuzu
+import pytest
+
+from dindex_store.common import DiscoveryIndex, EdgeType
+
+
+class DiscoveryIndexKuzu(DiscoveryIndex):
+
+    def __init__(self, directory: str = '', testing: bool = False):
+        # connect to Kuzu
+        # TODO: pass connection parameters as a config object
+        self.conn = kuzu.database('./test')
+        self.conn = kuzu.connection(self.db)
+
+        # Create schema
+        # FIXME: such schema would be also read from some external config file, although we dont need to do that
+        # FIXME: until we settle on what the schema must look like
+        self.conn.execute("""
+        CREATE NODE TABLE Column(id INT64, attr INT64, PRIMARY KEY (id))
+        """)
+        self.conn.execute("""
+        CREATE REL TABLE Edge(FROM Column TO Column, similar INT64)
+        """)
+
+        if testing:
+            return
+        
+        for filename in os.listdir(directory):
```

Comment:
I factored out the JSON loading to a different file. I think it'd be helpful to have a batch_loader here, but it should not assume the data will be in JSON files. It does not need to be the case. So this needs a bit of rethinking.